internal/cmd/config/config.go

```diff
@@ -546,13 +546,19 @@ func Parse(d string) (*Config, error) {
 			if !strutil.Printable(k) {
 				return nil, fmt.Errorf("Tag key %q contains non-printable characters", k)
 			}
+			if strings.Contains(k, ",") {
```

Comment:
Not that this is necessarily wrong, but what's the reason for this?

Reply:
Since no commas are allowed thru the CLI entry, this applies the same standards to tag entries from the config file. Otherwise you can input keys such as "k3,k4" if you input them this way

>   tags {
    key = ["val1", "val2"],
    key2 = ["val1", "val2"],
    "k3,k4" = ["vvv"],
  }